tests/sles4sap/cryptctl.pm

```diff
@@ -0,0 +1,195 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Start cryptctl server, encrypt partition
+# Maintainer: Alexei Tighineanu <atighineanu@suse.de>
+
+use base "sles4sap";
+use testapi;
+use strict;
+
+
+sub run {
+    my ($self) = @_;
+    # List of solutions is different between saptune in x86_64 and in ppc64le
+    # Will check whether test is running in ppc64le with the OFW variable
+    select_console 'root-console';
+    unless (script_output "rpm -ql cryptctl") {
+        script_run "zypper -n in cryptctl";
+    }
+    my $timeout = 7200;
+    # precompile regexes
+    my $access_password = qr/^Access password.*/m;
+    my $confirm_passwd  = qr/^Confirm access password.*/m;
+    my $pem             = qr/^PEM.*/m;
+    my $hostname        = qr/^Host name.*/m;
+    my $ipaddress       = qr/^IP address.*/m;
+    my $tcp             = qr/^TCP.*/m;
+    my $db              = qr/^Key database.*/m;
+    my $client_cert     = qr/^Should clients.*/m;
+    my $encryption_key  = qr/^Should encryption.*/m;
+    my $smtp            = qr/^SMTP.*/m;
+    my $question        = qr/^Would you.*/m;
+    my $running         = qr/^Key server is now running.*/m;
+    my $proc_cryptctl   = qr/.*\/usr\/sbin\/cryptctl.*/m;
+    my $hstname         = qr/.*host name.*/m;
+    my $port_num        = qr/.*number \[3737.*/m;
+    my $cert_keyserv    = qr/.*PEM-encoded CA certificate.*/m;
+    my $certkey2        = qr/.*PEM-encoded client certificate.*/m;
+    my $clientkey       = qr/.*PEM-encoded client key.*/m;
+    my $serv_passwd     = qr/^Enter key server's.*/m;
+    my $crypta          = qr/^Path of dir.*/m;
+    my $disk            = qr/^Path of disk.*/;
+    my $how_many        = qr/^How many.*/m;
+    my $ifthekey        = qr/^If the key.*/m;
+    my $doublecheck     = qr/^Please double check.*/m;
+
+    #my $HOSTNAME 	 = script_ouptut ('ifconfig | grep \"inet addr\" | grep 10. | cut -d \"\:\" -f2 | cut -d \" \" -f1');
```

Comment:
I suppose we can drop this commented line, right?

Reply:
yes. I will change that.